gap/DocumentationTree.gi

```diff
@@ -324,6 +324,12 @@ InstallMethod( Add, [ IsTreeForDocumentationNode, IsString ],
 end );
 
 ##
+InstallMethod( Add, [ IsTreeForDocumentation, IsString ],
+  function( tree, string )
+    Add( tree!.content, string );
+end );
```

Comment:
Please add a comment that explain when this method is used (I assume it is workspace specific, as I wasn't able to trigger it in my tests with packages).